.github/pull_request_template.md

```diff
@@ -0,0 +1,148 @@
+<!--
+# clinical-genomics-lund/nextflow_wgs pull request
+
+Thanks for contributing to the CMD nextflow_wgs pipeline. 
+-->
+# Description
+
+<!--
+Add a description of changes below and a description of the expected outcome.
+-->
+
+What is changed? What is the result of the changes?
+
+<!--
+    Are there any issue or issues that are linked to this PR? 
+    Use Github's syntax for closing keywords: `closes #1` or `fixes #1`
+-->
+
+fixes # (issue)
+
+## Type of change
+
+<!--
+    Major change counts as a change that breaks backward compatibility
+    Minor change is a substantial change that requires testing before deployment
+    Patch is a minor change like a bug fix, code comment/style fix, etc.
+    
+    Choose one and delete the remaining fields.
+-->
+
+- [ ] Major change 
+- [ ] Minor change
+- [ ] Patch
+
+# Checklist:
+
+<!--
+    The checklist below applies to all types of changes. 
+    
+    Do not hesitate to add your own items to the checklist if applicable.
+-->
+
+- [ ] My code follows the [style guidelines for NF pipelines at CMD](http://mtlucmds1.lund.skane.se/wiki/doku.php?id=nextflow&s[]=nextflow#code_style_at_cmd)
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] My changes generate no new warnings (Keep an eye on `.nextflow.log` !)
+- [ ] I have updated the CHANGELOG
+- [ ] The latest commit in the master branch is tagged
+      with the correct version number
+
+<!--
+    Select a checklist below  based on selection under # Type of change
+    and delete the sections that do not apply to this PR:
+-->
+
+## Major change
+
+- [ ] Stub run completes without errors or new warnings
+- [ ] `onco` run finishes without any new warnings/errors and the results can 
+       be loaded into scout
+- [ ] `wgs` single run finishes without any new warnings/errors and the results 
+       can be loaded into scout
+- [ ] `wgs` trio run finishes without any new warnings/errors and the results 
+       can be loaded into scout
+- [ ] At least one other person has reviewed and approved my code
+- [ ] I have made corresponding changes to the documentation
+
+## Minor change
+
+- [ ] Stub run completes without errors or new warnings
+- [ ] `onco` run finishes without any new warnings/errors and the results can 
+       be loaded into scout
+- [ ] `wgs` single run finishes without any new warnings/errors and the results 
+      can be loaded into scout
+- [ ] `wgs` trio run finishes without any new warnings/errors and the results 
+       can be loaded into scout
+- [ ] At least one other person has reviewed and approved my code
+- [ ] I have made corresponding changes to the documentation
+
+## Patch
+
+- [ ] Stub run completes without errors or new warnings
```

Comment:
I think the review-bullet should be here as well in some form. At least having someone glance over the code captures many silly errors. Should be little work it is a tiny change.

Reply:
Yes, I agree. It makes sense to leave the possibility open. Maybe it could ideally be structured such that it is a "recommended but skippable" step? I guess there will be many non-emergent patches which really should be reviewed.

I think we are on the same page here. I trust your judgement in how to articulate this!

Reply:
OK, maybe I am overly cautious here. I am thinking about patches to something like the wgs-pipeline. Doing small changes to a Python-based local script is probably less sensitive, and quick to fix if there are issues. Your approach above of letter the coder decide is perhaps a good balance!

Reply:
Here's an example of an unreviewed change made to the pipeline itself: https://github.com/Clinical-Genomics-Lund/nextflow_wgs/pull/77/files

Reply:
I believe it's reasonable to keep it optional for changes to Nextflow code or any of the utility scripts as in the linked example above. 

Reworded the comment a bit:

```suggestion
- [ ] Stub run completes without errors or new warnings
<!--
    Remove the comment markers from the section below if you believe that the patch needs to undergo review. 
    (If in doubt, then play it safe and request a review).
-->
<!--
- [ ] At least one other person has reviewed and approved my code
-->
```

Reply:
Yes, that is a good example. I totally agree that it is overkill to review that kind of change. Looks good!